src/lib/yang/schema.lua

```diff
@@ -673,7 +681,14 @@ function resolve(schema, features)
       end
       return ret
    end
-   function visit_type(node, env)
+   local function resolve_bases(bases, env)
```

Comment:
Please add a comment: "Resolve argument of "base" statements to identity fqid and collect in a list."